### PR TITLE
Support Unicode ≘≙≚≛≝≞≟

### PR DESCRIPTION
**Pull request by ronkok**
Supports Unicode characters ≘ ≙ ≚ ≛ ≝ ≞ ≟

### CODE REVIEW

**Comment by ronkok** [comment]
Visual confirmation:
stackrel

**Comment by edemaine** [comment]
By the way, are these supported in unicode-math?

**Comment by ronkok** [comment]
@edemaine  Good point about the braces. I've removed them. 

Re: `unicode-math`: Yes, it does support all of the characters in this PR, and applies `\mathrel` to each, as one would expect.

Visual confirmation on the latest commit:
stackrel

File: test/katex-spec.js [hunk 1]
```diff
@@ -2984,7 +2984,7 @@ describe("Unicode", function() {
     });
 
     it("should parse more relations", function() {
-        expect("⊂⊃⊆⊇⊏⊐⊑⊒⊢⊣⊩⊪⊸⋈⋍⋐⋑⋔⋙⋛⋞⋟⌢⌣⩾⪆⪌⪕⪖⪯⪰⪷⪸⫅⫆≘≙≚≛≝≞≟").toBuild();
+        expect("⊂⊃⊆⊇⊏⊐⊑⊒⊢⊣⊩⊪⊸⋈⋍⋐⋑⋔⋛⋞⋟⌢⌣⩾⪆⪌⪕⪖⪯⪰⪷⪸⫅⫆≘≙≚≛≝≞≟").toBuild();
```
[the remark this answers is not in the document]
Comment (ronkok) [reply test/katex-spec.js]: ⋙ was relocated, not removed. In PR #1273, @kevinbarabash  asked me to place ⋙ and ⋘ adjacent to each other in the test string, so I moved ⋙ from line 2987 to the end of line 2979. 

Unfortunately that move causes a merge conflict with this PR. I thought I had dealt with the conflict successfully. If there is something else I should do, let me know.

[the remark this answers is not in the document]
Comment (edemaine) [reply test/katex-spec.js]: Looking again, it seems all is well.  Merging.

**Pull request merged**